functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -139,7 +142,9 @@ function Install-DbaMaintenanceSolution {
         [switch]$InstallJobs,
         [string]$LocalFile,
         [switch]$Force,
+        [switch]$InstallParallel,
```

Comment:
Could you also add an example to the help for this new param please

Reply:
Example added, waiting to push/commit for further tweaks

`
    .EXAMPLE
        PS C:\> Install-DbaMaintenanceSolution -SqlInstance RES14224 -Database DBA -InstallParallel

        This will create the Queue and QueueDatabase tables for uses when manually changing jobs to use the @DatabasesInParallel = 'Y' flag`